src/main/java/com/unsri/ecommerce/domain/models/Inventory.java

```diff
@@ -21,7 +24,7 @@
     @Column(name = "fk_seller_id")
     private Integer fkSellerId;
 
-    @OneToMany(targetEntity = PhotoInventory.class)
+    @OneToMany(targetEntity = PhotoInventory.class, cascade = CascadeType.ALL)

```

Comment:
CascadeType.ALL digunakan untuk apa?

Reply:
Cascade type all ini sebenernya bukan bagian dari fitur create inventory tapi cascade type all ini digunakan untuk menghapus/update seluruh data dengan id yang kita pilih

Reply:
bisa kasih contoh? @aditiyars

Reply:
contohnya pada satu seller bisa memiliki banyak(lebih dari satu) foto untuk satu seller, kalo gak pake cascade bakal munculin transient exception (TransientObjectException occurs when you save an object which references another object that is transient (meaning it has the "default" identifier value, frequently null) and then flush the Session.)